{
  "Header": {
    "Original": "CVE-2019-20392 (An invalid memory access flaw is present in libyang before v1.0-r1 in  ...)",
    "Line": 293725,
    "ID": "CVE-2019-20392",
    "Description": "(An invalid memory access flaw is present in libyang before v1.0-r1 in  ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-3572-1}",
      "Line": 293726,
      "Type": "xref",
      "Bugs": [
        "DLA-3572-1"
      ]
    },
    {
      "Original": "[experimental] - libyang 1.0.167-1",
      "Line": 293727,
      "Type": "package",
      "Release": "experimental",
      "Package": "libyang",
      "Kind": "fixed",
      "Version": "1.0.167-1"
    },
    {
      "Original": "- libyang 1.0.176-1",
      "Line": 293728,
      "Type": "package",
      "Package": "libyang",
      "Kind": "fixed",
      "Version": "1.0.176-1"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=1793922",
      "Line": 293729,
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=1793922"
    },
    {
      "Original": "NOTE: https://github.com/CESNET/libyang/commit/32fb4993bc8bb49e93e84016af3c10ea53964be5",
      "Line": 293730,
      "Type": "NOTE",
      "Description": "https://github.com/CESNET/libyang/commit/32fb4993bc8bb49e93e84016af3c10ea53964be5"
    },
    {
      "Original": "NOTE: https://github.com/CESNET/libyang/issues/723",
      "Line": 293731,
      "Type": "NOTE",
      "Description": "https://github.com/CESNET/libyang/issues/723"
    }
  ]
}